client/src/components/container/user/Registeration/index.js

```diff
@@ -0,0 +1,85 @@
+import React ,{Component}from 'react';
+import axios from 'axios';
+import './style.css'
+import { Button ,Form,Input} from 'antd';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+
+
+class RegisterUser extends Component {
+  state = {
+    responseMsg:'',
+    redirect: false
+
+  }
```

Comment:
we don't need to store the response since we are not gonna use it, also I suggest the success property to indicate whether we are redirecting from the exit button (success set to false in this case) or redirecting after a successful registration (success set to true) 
```suggestion
  state = {
    success : false,
    redirect : false,
  };
```